public/js/lib/devtools/client/shared/components/reps/window.js

```diff
@@ -37,8 +37,8 @@ define(function (require, exports, module) {
       return "";
     },
 
-    getLocation: function (grip) {
-      return getURLDisplayString(grip.preview.url);
+    getDisplayValue: function (grip) {
+      return grip.preview.isGlobal ? "Global" : "Window";
```

Comment:
@janodvarko mind taking another look. Once this lands, I'd be happy to make an m-c patch and perhaps add some m-c tests.